lib/chef/cookbook/chefignore.rb

```diff
@@ -56,22 +54,32 @@ def parse_ignore_file
             ignore_globs << line.strip unless line =~ COMMENTS_AND_WHITESPACE
           end
         else
-          Chef::Log.trace("No chefignore file found at #{@ignore_file} no files will be ignored")
+          Chef::Log.debug("No chefignore file found. No files will be ignored!")
```

Comment:
This needs to be updated.

Reply:
I think this is actually correct - a `debug` level message seems reasonable.